app/Participant.php

```diff
@@ -19,6 +19,10 @@ class Participant extends Model
 		2 => 0.7,
 		3 => 0.5
 	];
+	const INFORMATION_CHANNEL = [
```

Comment:
Hernoemen naar `INFORMATION_CHANNEL_DESCRIPTION`?

Reply:
of INFORMATION_CHANNEL_DESCRIPTION_TABLE net zoals de INCOME_DESCRIPTION_TABLE